src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -45,6 +45,26 @@ public void storeMessage(Message message) {
     datastore.put(messageEntity);
   }
 
+    /** Stores FoodItem in Datastore. */
+  public void storeFood(FoodItem food) {
+    Entity foodentity = new Entity("FoodItem", food.getID().toString());
```

Comment:
In general for the datastore field names (here and below) we should standardize their naming convention. Here `name` is lowercase` but then down below `Date` is uppercase. I suggest that you name them the same as the camel case names in the class themselves.